core/build.gradle

```diff
@@ -305,9 +305,11 @@ task('verifyVersions') {
      * as possible after release. */
     Set<String> actualVersions = new TreeSet<>(
       indexCompatVersions
-        .findAll { false == it.unreleased }
+        .findAll { false == it.snapshot }
         .collect { it.toString() })
 
+    // TODO this is almost certainly going to fail on 5.4 when we release 5.5.0
```

Comment:
I believe so. I added this here because I noticed it here. I think it'd be nice to fix it in a separate PR though because it isn't really related to this.